src/function/algebra/simplify/wildcards.js

```diff
@@ -0,0 +1,19 @@
+import { isConstantNode, isFunctionNode, isOperatorNode, isParenthesisNode } from '../../../utils/is'
+export { isConstantNode, isSymbolNode as isVariableNode } from '../../../utils/is'
+
+export function isNumericNode (x) {
+  return isConstantNode(x) || (isOperatorNode(x) && x.isUnary() && isConstantNode(x.args[0]))
+}
+
+export function isConstantExpression (x) {
+  if (isConstantNode(x)) { // Basic Constant types
```

Comment:
For an additional constant type, there could be an input of variables that are treated like constants but are left unsubstituted, like `e` or `pi`.

Reply:
Interesting idea! (but, one step at a time?...)